lib/jsdom/browser/file/filereader.js

```diff
@@ -0,0 +1,202 @@
+"use strict";
+
+var DOMException = require("../../web-idl/DOMException");
+var EventTarget = require("../../living/generated/events/EventTarget");
+var core = require("../../level1/core");
+require("../../level2/core");
```

Comment:
This kind of requiring is not good (it easily leads to circular dependencies, due to jsdom's messy legacy codebase nature). Instead, use the style in living/, where the module.exports function takes `core` as an argument.